api/api.go

```diff
@@ -744,30 +741,54 @@ func CreateNamespaceWithOwnerID(cl *graphql.Client, name string, ownerID string)
 	return &response, nil
 }
 
-func getOrganization(cl *graphql.Client, organizationName string, organizationVcs string) (*GetOrganizationResponse, error) {
-	var response GetOrganizationResponse
+type GetOrganizationParams struct {
+	OrgName *string
+	VCSType *string
+	OrgID   *string
```

Comment:
Why use `*string` here? I suspect it's to use `nil` to mark the absence of the parameter?

If that's the case, `string` and its zero value `""` should be an adequate, idiomatic choice.

Reply:
Yes, I kinda ported this practice from the previous implementation, I'll remove it.